sympy/integrals/tests/test_transforms.py

```diff
@@ -115,22 +114,22 @@ def test_mellin_transform():
         (0, re(rho)), And(re(rho) - 1 < 0, re(rho) < 1))
     mt = MT((1 - x)**(beta - 1)*Heaviside(1 - x)
             + a*(x - 1)**(beta - 1)*Heaviside(x - 1), x, s)
-    assert mt[1], mt[2] == ((0, -re(beta) + 1), True)
+    assert mt[1], mt[2] == ((0, -re(beta) + 1), re(beta) > 0)
```

Comment:
Thanks (and to @jksuom ). This was a badly formed test that would have passed unless `mt[1]` was false.